sources/packages/web/src/views/institution/Login.vue

```diff
@@ -74,6 +75,7 @@ export default defineComponent({
     },
   },
   setup(props) {
+    sessionStorage.setItem(USER_LOGIN_TRIGGERED, "true");
```

Comment:
Should it not be added inside the login button action?
The idea is to know when the user initiated the login, right?

Reply:
The user loading the page vs the user clicking on the button are two different things.